buildpipeline/Core-Setup-Publish.json

```diff
@@ -314,10 +429,23 @@
     },
     "TeamName": {
       "value": "DotNetCore"
+    },
+    "VsoToolsRepo": {
```

Comment:
I'm not sure.  All of these new variables are used in the core-setup definition, not used by pipebuild - that means they should be non-`PB_`, right?